src/remarkable-ui-embeddables/theme/theme.types.ts

```diff
@@ -21,6 +22,9 @@ export type ThemeChartsLegendPosition = 'top' | 'right' | 'bottom' | 'left';
 export type Theme = {
   i18n: { language: string; translations: Resource };
   charts: {
+    overwritePieChart?: Partial<ChartOptions<'pie'>>;
+    overwriteDonutChart?: Partial<ChartOptions<'pie'>>;
+    overwriteDonutLabelChart?: Partial<ChartOptions<'pie'>>;
```

Comment:
The names are a bit misleading and they feel like a function name.
Maybe we can consider renaming to these?
```suggestion
    pieChartOverrides?: Partial<ChartOptions<'pie'>>;
    donutChartOverrides?: Partial<ChartOptions<'pie'>>;
    donutLabelChartOverrides?: Partial<ChartOptions<'pie'>>;
```

Reply:
done !